testsupport/metrics_server_assertions.go

```diff
@@ -13,3 +13,10 @@ func VerifyHostMetricsService(t *testing.T, hostAwait *wait.HostAwaitility) {
 	_, err := hostAwait.WaitForMetricsService()
 	require.NoError(t, err, "failed while waiting for 'host-operator-metrics' service")
 }
+
+// VerifyMemberMetricsService verifies that there is a service called `member-operator-metrics`
+// in the host namespace.
+func VerifyMemberMetricsService(t *testing.T, memberAwait *wait.MemberAwaitility) {
+	_, err := memberAwait.WaitForMetricsService()
```

Comment:
Should we do something beyond checking the existence of the service? eg. Is there an endpoint we can request to ensure the metrics service is working correctly?

Reply:
well, I'm going to use the service for the perf tests, but to make a request to an endpoint, I also need to create a route since the tests don't run within the cluster.

Reply:
oh ok, I guess any problems with the metrics service would be discovered by the perf tests then so that should be good enough